source/_docs/configuration/basic.markdown

```diff
@@ -14,6 +14,7 @@ homeassistant:
   longitude: 117.22743
   elevation: 430
   unit_system: metric
+  currency: "USD",
```

Comment:
Personally I would like the symbol but 🤷‍♂️ 
```suggestion
  currency: "$",
```

Reply:
Fine by me :)